LondonBritishLibrary/orient/BLorient5268.xml

```diff
@@ -0,0 +1,400 @@
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="158v" to="159r"/>
+                                    <desc type="GuestText">Magical Prayer against charms and diseases</desc>
+                                    <q xml:lang="en">ስፍሐዝዝር፡ ፒሮስ፡ አልክምኑን፡ ኤፍታሔ፡ ስፍሐዝዝር፡ ፒሮስ፡ አልክምኑን፡ ኤፍታሔ፡ ፍታሕ፡ ኵሎ፡ ሥራየ፡ ወኵሉ፡ መቅጹተ፡ ዘተገብር፡ 
+                                        እምኀበ፡ አረሚ፡ አው፡ እምክርቲያን፡ <gap reason="ellipsis"/></q>
+                                </item>
+                            </list>
+                        </additions>           
+                    </physDesc>  
+                    </msPart>
+                    <msPart xml:id="p2">
+                        <msIdentifier></msIdentifier>
+                        <msContents><summary/>
+                            <msItem xml:id="p2_i1">
+                                <locus target="#160"/><locus target="#164"/>
+                                <title>Fragments of prayers</title>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent><measure unit="leaf">2</measure></extent>
+                                    <condition key="deficient">A part of <locus target="#160"/> is torn off and <locus target="#164"/> has been badly
+                                        damaged by a corrosive liquid.</condition>
+                                </supportDesc>
+                                <layoutDesc>
+                                    <layout columns="2"/>
+                                </layoutDesc>
+                            </objectDesc>  
+                        </physDesc>
+                    </msPart>
+                     <msPart xml:id="p3">
+                         <msIdentifier></msIdentifier>
+                         <msContents><summary/>
+                            <msItem xml:id="p3_i1">
+                                <locus from="161" to="162"/>
+                                <title ref="LIT4711Mastabqwe" type="incomplete">Deprecatory prayers</title>
+                                <incipit xml:lang="gez"><title>መስተበቍዕ፡</title></incipit>
+                                <note>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">36</citedRange>
+                                    </bibl> several deprecatory prayers (mastabqʷǝʿāt) similar to those attested in 
+                                    <ref type="mss" corresp="0000000000000000000000000000000">BnF, éth. 309</ref> 
```

Comment:
yes you can refer, the ID should follow the pattern for other Éthiopien mss, we have not gotten to the Griaule collection yet, but surely you can also provide the actual folio numbers `<ref type="mss" corresp="BNFet309">BnF, éth. 309, ff. 72-75</ref>`

Reply:
OK. But would it not be better to create a stub for BnF, éth. 309? In this case, BNFet309 would actually exist at the time this record is merged.